src/test/java/g0501_0600/s0572_subtree_of_another_tree/SolutionTest.java

```diff
@@ -0,0 +1,29 @@
+package g0501_0600.s0572_subtree_of_another_tree;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com_github_leetcode.TreeNode;
+import com_github_leetcode.TreeUtils;
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+
+class SolutionTest {
+    @Test
+    void isSubtreeFound() {
+        TreeNode treeNode =
+                TreeUtils.constructBinaryTree(new ArrayList<>(Arrays.asList(3, 4, 5, 1, 2)));
+        TreeNode subTree = TreeUtils.constructBinaryTree(new ArrayList<>(Arrays.asList(4, 1, 2)));
+        assertThat(new Solution().isSubtreeFound(treeNode, subTree), equalTo(false));
```

Comment:
It should be true

![image](https://user-images.githubusercontent.com/348496/148198980-a65802df-c91e-49a1-a8cf-c75cd880648b.png)